lib/View.php

```diff
@@ -57,4 +57,22 @@ public function draw($template)
         extract($this->_variables);
         include $path;
     }
+
+    /**
+     * echo script tag incl. SRI hash for given script file
+     *
+     * @access private
+     * @param  string $file
+     * @param  bool   $async should it execute ASAP or only after HTML got parsed
+     */
+    private function _scriptTag($file, $async = true)
+    {
+        $sri = array_key_exists($file, $this->_variables['SRI']) ?
+            ' integrity="' . $this->_variables['SRI'][$file] . '"' : '';
+        $suffix = preg_match('#\d.js$#', $file) == 0 ?
```

Comment:
RegEx always could have a negative performance effect (in theory)… (Note RegEx is usually the slowest thing to implement pattern matching due to it's potential complexity) as this is executed inline for each paste access/template…

Could we maybe "outsource" this cost to some "config validation" that is only run once ("at startup" I would say if this were not a PHP application, because obviously it does not have this hmm).

Also code-readability-wise I don't quite get:
* what are these hash signs for? … ahhhh looking at PHP dic it's sed(-like) syntax for allowing slashes okay, fair enough, just me not knowing PHP commands here
* then, however, it matching `.js` ending files, okay, but why the `\d`, is it really relevant to only match JS files that end with ~~a~~ one digit aka number? Why not all files? 
* also, is not the logic inversed here: When finds 0 matches aka does not match a JS file, then append the version? Maybe inverse the if here, so the "real" "true" condition comes first?
* note one may want to mention the reason (I'm fine with a commend) for this: adding a ~~cache breaker~~ [cache buster](https://stackoverflow.com/questions/9692665/cache-busting-via-params) (that's apparently the correct term) – or just generally extract it into a function like "getJsSuffix"?

Reply:
4b6c835 should address this